packages/components/src/__actions__/LinkButton/v3/_docs/LinkButton--api-specification.mdx

```diff
@@ -0,0 +1,252 @@
+import { Canvas, Meta, Controls, ArgTypes, DocsStory } from "@storybook/blocks"
+import { ResourceLinks, KAIOInstallation } from "~storybook/components"
+import { LinkTo } from "../../../../../../../docs/components/LinkTo"
+import * as exampleStories from "./LinkButton.doc.stories"
+
+<Meta title="Actions/LinkButton/LinkButton (v3)/API Specification" />
+
+# LinkButton API Specification (v3)
+
+Updated Nov 26, 2024
+
+<ResourceLinks
+  sourceCode="https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/__actions__/LinkButton/v3"
+  figma="https://www.figma.com/design/eZKEE5kXbEMY3lx84oz8iN/%F0%9F%92%9C-Heart-UI-Kit?node-id=1929-17364"
+  designGuidelines="/?path=/docs/actions-linkbutton-linkbutton-v3-usage-guidelines--docs"
+/>
+
+<KAIOInstallation exportNames={["LinkButton" ]} family="actions" version="3" />
+
+## Overview
+
+`LinkButton` allows users to navigate to another page or resource. It shares the same visual styles and interaction states as the <LinkTo pageId="actions-button-button-v3-usage-guidelines--docs">Button</LinkTo> component, but is intended for navigational purposes and downloading documents.
+
+The following example and table showcases the essential props that enable the core functionality of `LinkButton`. For the remaining suite of API options refer to [this section](#additional-api-options).
+
+<Canvas of={exampleStories.Playground} />
+
+<Controls of={exampleStories.Playground} include={["className", "children", "href", "target", "download", "onPress", "routerOptions", "hasHiddenLabel",  "size", "variant", "icon", "iconPosition", "isFullWidth", "isDisabled" ]} />
+
+## API
```

Comment:
Perhaps make `API` the heading for the actual component props, and give this section a heading of `Setup`.
Alternatively, we could move the setup into a different doc since it's quite long.

Reply:
Its long but I don't think it belong in another doc yet since it only relates to this component. Maybe when Link comes in or if we do set up an external LinkProvider the FE services repo but it would just feel unattached anywhere else / is quite important to the feature of this component 🤔

Reply:
I suggest you alter the headings a bit then :)

Looking at it again, many of these subheadings are probably okay to be under `Overview`, then `Set up` from `Client side routing`, then `API` from `Variants`.